### PR TITLE
Feat: Store & redux dev tool extension added

### LINKED ISSUE #4: Integrate redux dev tools extension
- integrate redux dev tools extension 

https://github.com/zalmoxisus/redux-devtools-extension

This has be to be done after https://github.com/samundra/whoowesme/issues/3 has been completed.

### PR DESCRIPTION
**Pull request by pradeepravi-cse**
closes: #4

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
+  overflow: "auto",
+  height: "auto",
+  minHeight: "400px"
 } as React.CSSProperties;
 
 const navigationStyle = {
-  width: '1024px',
-  margin: '0 auto',
+  width: "1024px",
+  margin: "0 auto"
```
Comment (samundra) [inline src/App.tsx]: Lets use single quote for convention as we already have single quote as convention.

**Comment by samundra** [comment]
Changes look good to me. Lets change all double quotes to single quote.

File: src/App.tsx (hunk 1 above)
Comment (pradeepravi-cse) [reply src/App.tsx]: Updated :+1:

**Review by samundra — APPROVED** [review]

**Comment by samundra** [comment]
@pradeepravi-cse Thank you 👍

**Pull request merged**